opentreemap/treemap/lib/object_caches.py

```diff
@@ -69,6 +69,7 @@ def increment_adjuncts_timestamp(instance):
     from treemap.models import Instance
     Instance.objects.filter(pk=instance.id)\
                     .update(adjuncts_timestamp=F('adjuncts_timestamp') + 1)
+    instance.refresh_from_db()
```

Comment:
@maurizi, on further reflection I think it would be safer to do:
``` python
    qs = Instance.objects.filter(pk=instance.id)
    qs.update(adjuncts_timestamp=F('adjuncts_timestamp') + 1)

    # Fetch updated timestamp so it's not stale
    instance.timestamp = qs[0].timestamp
```

Otherwise we risk clobbering values in `instance` that haven't been saved yet.